wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -54,9 +58,12 @@ fi
 MACHINE_TYPE=$(uname -m)
 
 mkdir -p ${CHE_DIR}
-${SUDO} mkdir -p /projects
-${SUDO} sh -c "chown -R $(id -u -n) /projects"
-${SUDO} chmod 755 /projects
+
+if is_current_user_sudoer; then
+    ${SUDO} mkdir -p /projects
+    ${SUDO} sh -c "chown -R $(id -u -n) /projects"
+    ${SUDO} chmod 755 /projects
+fi
```

Comment:
in che-server image we updated only directory permissions

would it work as well if we only change permissions of folder but not the owner ?
(it can make the operation faster)

Reply:
Do you mean this? https://github.com/eclipse/che/blob/master/dockerfiles/che/Dockerfile#L43

Reply:
@eivantsov yes

Reply:
I was thinking replacing 2 instructions chown + chmod with the only find + exec